demo/vl-cookie-consent.html

```diff
@@ -7,101 +7,96 @@
 
-    <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-button/dist/style.css"/>
+    <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-button/dist/style.css" />
 </head>
 
 <body>
     <h1>vl-ui-cookie-consent demo</h1>
-    
+
     <div>
         <h2>Cookie-consent</h2>
         <pre>
             <code class="line-numbers language-markup">
-                &lt;vl-cookie-consent id="consent-1" data-vl-auto-open-disabled&gt;&lt;/vl-cookie-consent&gt;
+                &lt;vl-cookie-consent data-vl-auto-open-disabled&gt;&lt;/vl-cookie-consent&gt;
             </code>
         </pre>
         <div class="demo">
-            <vl-cookie-consent id="cookie-consent-1" data-vl-auto-open-disabled></vl-cookie-consent>
-            <button id="button-open-cookie-consent-1" is="vl-button" onclick="document.querySelector('#cookie-consent-1').open();">Open
-                cookie-consent</button>
+            <vl-cookie-consent id="cookie-consent" data-vl-auto-open-disabled></vl-cookie-consent>
+            <button id="button-open-cookie-consent" is="vl-button" onclick="document.querySelector('#cookie-consent').open();">Open cookie-consent</button>
         </div>
     </div>
 
     <div>
         <h2>Cookie-consent zonder functionele opt-in</h2>
         <pre>
             <code class="line-numbers language-markup">
-                &lt;vl-cookie-consent id="consent-2" data-vl-auto-open-disabled data-vl-auto-opt-in-functional-disabled&gt;&lt;/vl-cookie-consent&gt;
+                &lt;vl-cookie-consent data-vl-auto-open-disabled data-vl-auto-opt-in-functional-disabled&gt;&lt;/vl-cookie-consent&gt;
             </code>
         </pre>
         <div class="demo">
-            <vl-cookie-consent id="cookie-consent-2" data-vl-auto-open-disabled data-vl-auto-opt-in-functional-disabled>
-            </vl-cookie-consent>
-            <button id="button-open-cookie-consent-2" is="vl-button" onclick="document.querySelector('#cookie-consent-2').open();">Open
-                cookie-consent</button>
+            <vl-cookie-consent id="cookie-consent-without-functional-opt-in" data-vl-auto-open-disabled data-vl-auto-opt-in-functional-disabled></vl-cookie-consent>
+            <button id="button-open-cookie-consent-without-functional-opt-in" is="vl-button" onclick="document.querySelector('#cookie-consent-without-functional-opt-in').open();">Open cookie-consent</button>
         </div>
     </div>
 
     <div>
         <h2>Cookie-consent met extra opt-in</h2>
         <pre>
             <code class="line-numbers language-markup">
-                &lt;vl-cookie-consent id="consent-3" data-vl-auto-open-disabled&gt;
+                &lt;vl-cookie-consent data-vl-auto-open-disabled&gt;
                     &lt;vl-cookie-consent-opt-in data-vl-label="Sociale media" data-vl-description="Beschrijving van de sociale media cookies."&gt;&lt;/vl-cookie-consent-opt-in&gt;
                 &lt;/vl-cookie-consent&gt;
             </code>
```

Comment:
idem... maar ook hier, als hij er bewust niet bijgezet is, is dat voor mij ook niet echt nodig.

Reply:
Deze buttons zijn er inderdaad bewust niet bijgezet omdat ze buiten het openen niet echt logica bevatten en de code voorbeelden alleen zouden vervuilen.